tutorials/camkes-vm-kernel-module/module_minimal/qemu-arm-virt/devices.camkes

```diff
@@ -0,0 +1,45 @@
+<!--
+  Copyright 2021 Michael Neises
+
+  SPDX-License-Identifier: BSD-2-Clause
+-->
```

Comment:
Should use CAmkES comment (same as C style)

Reply:
I've made the suggested edit.